core/src/components/router-outlet/router-outlet.tsx

```diff
@@ -19,6 +19,9 @@ import type {
 import type { RouterOutletOptions, SwipeGestureHandler } from '../nav/nav-interface';
 import type { RouteID, RouterDirection, RouteWrite, NavOutlet } from '../router/utils/interface';
 
+/**
+ * @virtualProp {"ios" | "md" | "ionic"} theme - The theme determines the visual appearance of the component.
+ */
```

Comment:
This component also doesn't have a virtualProp for mode. Should it?

Reply:
This component has an explicit property for `mode` on L45.